### DISCUSSION TITLE
Breakpoints hit on wrong line in source while debugging tests in VSCode

### DISCUSSION THREAD

**Discussion by p-bakker** [vitest-dev/vscode#540] [Q&A]
Maybe not the best description, but the problem I'm facing is that when hitting a breakpoint while debugging tests, the line in the editor on which the debugger breaks is not the proper line

I'm just authoring plain TypeScript files, using the Vitest integration for VSCode, with the latest TypeSCript/VSCode/Vitest extension etc

Is this a known issue or some special setup required to make this work properly?

**Comment by sheremet-va** [comment]
It should generally work correctly, but there are some known issues with exports: https://github.com/vitest-dev/vitest/issues/5569

**Reply by p-bakker** [reply]
Mmm, dunno why I'm running into this issue then, as my setup if really simple and breakpoint hitting is just all over the place...
